src/components/Footer/index.jsx

```diff
@@ -0,0 +1,64 @@
+import facebook from "../../assets/facebook.svg";
+import twitter from "../../assets/twitter.svg";
+import youtube from "../../assets/youtube.svg";
+import instagram from "../../assets/instagram.svg";
+import styles from "./index.module.css";
+
+const snsLists = [
```

Comment:
상수값은 대문자로 네이밍해주는 게 일반적인 거 같아요!
```suggestion
const SNS_LISTS = [
```